src/main/resources/scipooldev.properties

```diff
@@ -1,4 +1,5 @@
 SYNAPSE_OAUTH_CLIENT_ID=100055
+SYNAPSE_OAUTH_CLIENT_SECRET=/synapse-login-scipooldev/SynapseOauthClientSecret
```

Comment:
The way the login app works today is that "SYNAPSE_OAUTH_CLIENT_SECRET" *is* the key that you are to use in SSM.  If you want to use a different key name then a change to the code is necessary.